src/ContainerDeleteModal.jsx

```diff
@@ -6,22 +6,22 @@ import * as utils from './util.js';
 const _ = cockpit.gettext;
 
 const ContainerDeleteModal = (props) => {
-    return (
-        <div>
-            <Modal isOpen={props.selectContainerDeleteModal} fade={false} >
-                <ModalHeader>
-                    {cockpit.format(_("Please confirm deletion of $0"), utils.truncate_id(props.containerWillDelete.ID))}
-                </ModalHeader>
-                <ModalBody>
-                    {_("Deleting a container will erase all data in it.")}
-                </ModalBody>
-                <ModalFooter>
-                    <Button color="secondary" onClick={props.handleCancelContainerDeleteModal}>Cancel</Button>
-                    <Button color="danger" onClick={props.handleRemoveContainer}>{_("Delete")}</Button>{' '}
-                </ModalFooter>
-            </Modal>
-        </div>
-    );
+	return (
+		<div>
+			<Modal isOpen={props.selectContainerDeleteModal} fade={false} >
+				<ModalHeader>
+					{cockpit.format(_("Please confirm deletion of $0"), utils.truncate_id(props.containerWillDelete.ID))}
+				</ModalHeader>
+				<ModalBody>
+					{_("Deleting a container will erase all data in it.")}
+				</ModalBody>
+				<ModalFooter>
+					<Button color="secondary" onClick={props.handleCancelContainerDeleteModal}>Cancel</Button>
+					<Button id="container-delete-toggle-btn" color="danger" onClick={props.handleRemoveContainer}>{_("Delete")}</Button>{' '}
+				</ModalFooter>
+			</Modal>
+		</div>
+	);
```

Comment:
Can you please fix your editor to not destroy the indentation in that way? 4 spaces, no tabs, please. And mass-replace `\t` with four spaces in the other files here, too.